This moves the documentation out of /usr/local/share/cfengine/{doc,html} into
/usr/local/shared/doc/cfengine to keep it inline with the rest of the ports.

--- doc/Makefile.am.orig	Fri Dec  1 02:24:39 2000
+++ doc/Makefile.am	Thu Sep 19 08:39:57 2002
@@ -2,7 +2,7 @@
 info_TEXINFOS = cfengine-Reference.texinfo cfengine-Tutorial.texinfo
 
 # Info used in building and installing HTML files
-htmldir = $(pkgdatadir)/html
+htmldir = $(datadir)/doc/cfengine
 htmlfiles = cfengine-Reference.html cfengine-Tutorial.html 
 html_DATA = $(htmlfiles) cf-security.html
 
@@ -14,21 +14,22 @@
 # then uncomment the reference to ps2
 psfiles = $(ps1)	# $(ps2)
 
-psdir = $(pkgdatadir)/doc
+psdir = $(datadir)/doc/cfengine
 ps_DATA = $(psfiles)
 
 # Make sure these get distributed with everything else.
 EXTRA_DIST =								\
 	texinfo.tex							\
-	cf-security.html						\
+	$(html_DATA)							\
+	$(ps_DATA)							\
 	locks.latex							\
 	paper1.latex							\
 	paper2.latex
 
-SUFFIXES = html latex
+SUFFIXES = .html .latex
 
 %.html : %.texinfo
-	${MAKEINFO} --html $<
+	${MAKEINFO} --html --no-split $<
 
 %.dvi : %.latex
 	${LATEX} $<
